source-container-build/app/source_build.py

```diff
@@ -331,6 +340,13 @@ def _find_prefetch_source_archives():
         shutil.copy(src, dest)
         sib_dirs.extra_src_dirs.append(f"{prepared_sources_dir}/{src_dir}")
 
+    for root, filename in _find_prefetch_srpm_archives():
+        next(source_counter)
```

Comment:
IIRC the source_counter is used for the names of non-SRPM sources (those that go into the `extra_src_dirs`. There's no need to increment the counter when we encounter an SRPM.